tests/cleaner_tests/full_report_run.py

```diff
@@ -0,0 +1,86 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import json
+import re
+
+from avocado.utils import process
+from sos_tests import StageTwoReportTest
+
+
+class FullCleanTest(StageTwoReportTest):
+    """Run an unrestricted report execution through sos clean, ensuring that
+    our obfuscation is reliable across arbitrary plugin sets and not just the
+    'main' plugins that tend to collect data needing obfuscation
+
+    :avocado: tags=stagetwo
+    """
+
+    sos_cmd = '-v --clean'
+    sos_timeout = 600
```

Comment:
Will 10 minutes suffice? As I see some CI jobs taking 10m now (well they run various tests and not just one, but afaik much limited)?

Either way, we can increase it any time once we realize it is too low.

Reply:
Yeah, the 10m is the whole job execution time. With re-ordering the tests, the `clean` tests are executing much faster due to not consuming the garbage from the stage 2 logs test. But absolutely we can adjust this as we need going forward.